### PR TITLE
Adds a Stations and Syndicates rulebook and monster manual

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Adds a Stations and Syndicates rulebook, a table top RPG in a similar vien to D&D but with a shorter and more adaptable ruleset for the limited interaction and short time limit of SS13 rounds.

All the total changes are :
Added a rulebook, 
Added a basic monster manual to replace the Creature Conspectus with a couple SS13 critters and monsters
Added both of the above to most maps as well as added the books to 2 vending machines
Renamed the existing Creature Conspectus to have the "Original Edition" tag to differentiate it from the version I added.
Rearranged some nerd dungeon items to accomodate the new books + removed a TV in Cog2's dungeon since it broke the general flow of the room.

If you have any feedback or questions regarding the system feel free to leave a comment

## Why's this needed? 
It's a system I've tested with someone else on a live roundd and it works. Traditional D&D doesnt flow that well in an atmosphere like SS13 and after having managed to find another system that does and adapting it to fit the setting of SS13, I thought it'd be worth adding.

I'm happy to demonstrate a session in game with any interested volunteers if there's any concern for the actual functionality of the system. I'll upload a .txt version of both the manual and rulebook below.

## Changelog

```
(u)Carbadox
(*)Added an "official" Stations and Syndicates rulebook to the game, the SS13 version of Dungeons and Dragons. It's been made to accommodate SS13 virtual environment with a much light ruleset. It comes along with a basic monster manual. Check out your local nerd dungeons for a copy or a book/gaming vendor.
```

### CODE REVIEW

**Comment by Carbadox** [comment]
[Creature Conspectus.txt](https://github.com/goonstation/goonstation/files/5836452/Creature.Conspectus.txt)

[stations and syndicates.txt](https://github.com/goonstation/goonstation/files/5836455/stations.and.syndicates.txt)

**Comment by Gannets** [comment]
Always nice to encourage players to actually run s&s

**Review by flrsh — APPROVED** [review]
Nice!

File: code/obj/item/book.dm [hunk 1]
```diff
@@ -973,6 +973,138 @@
...
 
+/obj/item/paper/book/monster_manual_revised
+	name = "Creature Conspectus Revised Edition"
+	desc = "A large book detailing many creatures of myth and legend for the tabletop RPG, Stations and Syndicates."
+	icon_state = "book3"
+
+	info = {"
+<body>
+<h2>Note for the Game Master</h2>
+<p>This bestiary is written to be flexible. As such, feel free to consider the stats as pure guidelines. If you think a creature is too strong, feel free to lower it's stats. Same applies vice versa. Certain details have been omitted, for example the saving throws listed in monster abilities are absent to allow the GM to tweak them at will. Vulnerabilities are also open to interpretation.
```
Comment (flrsh) [inline code/obj/item/book.dm:984]:
```suggestion
<p>This bestiary is written to be flexible. As such, feel free to consider the stats as pure guidelines. If you think a creature is too strong, feel free to lower its stats. Same applies vice versa. Certain details have been omitted, for example the saving throws listed in monster abilities are absent to allow the GM to tweak them at will. Vulnerabilities are also open to interpretation.
```

File: code/obj/item/book.dm [hunk 2]
```diff
@@ -973,6 +973,138 @@
...
+<b>Killer Tomato</b><br>
+Type : Plant<br>
+Size : Small<br>
+DC To Hit : 5<br>
+Resolve : 1<br>
+Damage per attack : 1 (Bite)<br>
+Vulnerability : Slashing / Blunt<br>
+<br>
+<b>Messy Death</b><br>
+<i>On death, the creature's body will splash it's boiling inner fluids in a random direction, dealing one resolve damage.</i><br>
```
Comment (flrsh) [inline code/obj/item/book.dm:997]:
```suggestion
<i>On death, the creature's body will splash its boiling inner fluids in a random direction, dealing one resolve damage.</i><br>
```

File: code/obj/item/book.dm [hunk 3]
```diff
@@ -973,6 +973,138 @@
...
+<br>2 – To take an action, describe what you want to do and then – if the GM asks – make a success check. If you fail, the GM can make a reaction against you.<br>
+<br><b>Success Check or Reaction = 1d20 +  Attribute +  Archetype vs Difficulty Class</b><br>
+<br>3 – If something attacks or acts against you, make a success check to defend.<br>
+<br>4 – If you suffer harm, you lose 1 resolve. If you deal harm, your target loses 1 resolve.<br>
+<br>5 – You can spend 1 resolve to roll with advantage or attempt an extraordinary feat.<br>
+<br>6 – If you have 0 resolve, you are knocked out. You gain 1 resolve when you finish a long rest.<br>
+<br>7 – You can carry 3 significant items at once.<br>
+<br>8 – When the quest is resolved, the game ends! If you survive, gain a perk. Choose either +1  Attribute point, +1 Archetype point or +1 maximum resolve.<br>
+
+This ruleset is entirely based upon the Quick Quest system, with minor changes. The source material is made by GiffyGlyph, which can be found at https://giffyglyph.com/quickquest/
```
Comment (flrsh) [inline code/obj/item/book.dm:1104]:
```suggestion
This ruleset is entirely based upon the Quick Quest system, with minor changes. The source material is made by GiffyGlyph, which can be found <a href="https://giffyglyph.com/quickquest" target="_blank">here</a>.
```

**Review by flrsh — APPROVED** [review]

**Pull request merged**